views/lexicon/forms/compound.html.twig

```diff
@@ -301,15 +304,17 @@
             'id': 'comp-8',
             'help': 'For example: 25 12 1980',
             'label': 'Date of birth',
-            'error': ['Do this', 'Do this too'],
+            'error': ['Invalid day of birth entered', 'Invalid year of borth entered'],
+            'class': 'form__group--partial-state',
             'inputs': [
                 {
-                    'class': 'form__control-col--1',
+                    'class': 'form__control-col--1 has-error',
```

Comment:
We shouldn't need to explicitly add the error class if `has_error` is passed, the following example on L:341 doesn't...

Reply:
Yes, `has_error` isn't connected to styling.

Reply:
I mean if the input uses `has_error: true`, then the `has-error` class should be applied automatically?